### ISSUE TITLE
Suggestion: Use CompatLayer for binary compatible versions

### ISSUE THREAD

**Issue by D3nnis3n** [PrinceOfAmber/Cyclic#369]
I know i will most likeley fail but as i love this mod i just try it.

I'll suggest looking into: https://minecraft.curseforge.com/projects/compatlayer?gameCategorySlug=mc-mods&projectID=253729

This makes it possible for loads of mods out there to write one single mod, that works on both 1.11.2 and 1.10.2 if you also use the correct CompatLayer for the respective minecraft version.

*blinkingwithtearedeyes*

**Comment by PrinceOfAmber** [comment]
Yeah its worth looking into.  I heard its lots of work but we'll see.

I guess lots of big modpacks are still on 1.10 as well.

I made a poll on my twitter to try and see how many people still want 1.10.2, so vote and tell all your friends to as well https://twitter.com/lothrazar/status/860894740396392448

**Comment by D3nnis3n** [comment]
I'd like to vote - but i haven't got twitter and will not register at any social media platform.
Please add my vote in your mind *g*

**Comment by D3nnis3n** [comment]
Well, at least 35% of your users are still using  a 1.10.2 version and i believe it's much more, as the quota is hard to know when you release 1.11.2 versions more often. Nearly all players of techmods - and especially to them your mod fits good - have no chance to upgrade to 1.11.2 as mods like enderio, mekanism, and the likes are not even near 1.11.2 yet. Also, in knowledge of 1.12 being yet another big breaking update there are modders concentrating on that already and skipping 1.11.2 - i see 1.11.2 as one of those 'skipped' versions like 1.8 and 1.9 for big mods. I really hope my 26 mods of 152 that did not upgrade will do, but that is not going to happen soon. Also, don't focus the numbers - the 125 mods make 10% of the features in my modpack - the rest comes from the mods not updated :D

**Comment by PrinceOfAmber** [comment]
.. the bottom line is i just dont have the time. if i had a few dozen hours i would fill everything in for 1.10, hell i could even port everything to 1.710 i i had a few spare days.  I just dont have the time, im just one person.

But most modders and modpacks are moving to 1.11 and now 1.12 is coming out, i know Darkosto's latest pack is already in the works for 1.11 and he wants to jump to 1.12 if possible.  Since this issue was made, mekanism is already on 1.11 now,  Thermal foundation and tinkers are being updated,  and more will follow.  


 Also, compat layer will be abandoned for 1.12 https://www.reddit.com/r/feedthebeast/comments/6gl0i8/probably_giving_up_on_compat_layer_for_112/ 


Bottom line: Yes i will still be supporting 1.10.2 for any thing broken, any bugfixes, it will still get that support.
